src/components/TagItem.vue

```diff
@@ -79,6 +90,9 @@ export default {
 		}
 	},
 	methods: {
+		deleteTag() {
+			this.$emit('delete-tag', this.tag)
```

Comment:
```suggestion
			this.$emit('deleteTag', this.tag)
```

Recommendation from https://v3.vuejs.org/guide/component-custom-events.html#event-names